copilot/copilot/app.py

```
@@ -0,0 +1,34 @@
+async def deploy_app(name: str, image: str, replicas: int = 1):
+    v1 = client.AppsV1Api()
+
+    deployment = client.V1Deployment(
+        metadata=client.V1ObjectMeta(name=name),
+        spec=client.V1DeploymentSpec(
+            replicas=replicas,
+            selector={"matchLabels": {"app": name}},
+            template=client.V1PodTemplateSpec(
+                metadata={"labels": {"app": name}},
+                spec=client.V1PodSpec(
+                    containers=[client.V1Container(name=name, image=image)]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)